docs/api.yaml

```diff
@@ -12264,7 +12264,7 @@ components:
       properties:
         name:
           type: string
-          example: myfile.mp3
+          example: myfile.png
```

Comment:
What was wrong with this file being a PNG?

Reply:
I think either way would be fine, but I think the consistency doesn't hurt.